doc/7-systems/README.md

```diff
@@ -1,56 +1,813 @@
+<!-- -->
+- **Commander X16 (VERA only)**: [VERA](vera.md)
+<!-- -->
+- and many, many more!
+
+
+
+# chips
+
+this is the full list of chips that Furnace supports.
+
+- [2A03](nes.md)
 - [Amiga](amiga.md)
-- [AY-3-8910](ay8910.md)
-- [Microchip AY8930](ay8930.md)
-- [Bubble System WSG](bubblesystem.md)
-- [Casio PV-1000](pv1000.md)
-- [Commodore 64](c64.md)
-- [Commodore PET](pet.md)
-- [Commodore VIC-20](vic20.md)
-- [Generic PCM DAC](dac.md)
-- [Famicom Disk System](fds.md)
+- [AY-3-8910, AY-3-8914, YM2149(F), 5B](ay8910.md)
+- [AY8930](ay8930.md)
+- [MOS 6581, MOS 8580](c64.md)
+- [ES5506](es5506.md)
+- [SCC](scc.md)
+- [FDS](fds.md)
 - [Game Boy](game-boy.md)
-- [Konami K007232](k007232.md)
-- [Konami K056320](k056320.md)
-- [Konami SCC](scc.md)
-- [Konami VRC6](vrc6.md)
-- [Atari Lynx](lynx.md)
-- [Namco 163](n163.md)
-- [Namco WSG](namco.md)
-- [NES](nes.md)
-- [Nintendo MMC5](mmc5.md)
-- [OKI MSM5232](msm5232.md)
-- [OKI MSM6258](msm6258.md)
-- [OKI MSM6295](msm6295.md)
-- [PC Engine/TurboGrafx-16](pce.md)
+- [Generic PCM [DAC](dac.md)](dac.md)
+- Irem GA20
+- [Konami WSG](bubblesystem.md)
+- [K007232](k007232.md)
+- [K053260](k056320.md)
+- [Lynx](lynx.md)
+- [MMC5](mmc5.md)
+- [MSM5232](msm5232.md)
+- [MSM6258](msm6258.md)
+- [MSM6295](msm6295.md)
+- [N163](n163.md)
+- [Namco WSG, C15, C30](namco.md)
+- [HuC6280](pce.md)
 - [PC Speaker](pcspkr.md)
-- [Philips SAA1099](saa1099.md)
-- [Pokémon mini](pokemini.md)
-- [Capcom QSound](qsound.md)
-- [Ricoh RF5C68](ricoh.md)
+- [PET](pet.md)
+- [Pokémon Mini](pokemini.md)
+- [POKEY](pokey.md)
+- Pong
+- [PV-1000](pv1000.md)
+- [QSound](qsound.md)
+- [RF5C68, RF5C164](ricoh.md)
+- [SAA1099](saa1099.md)
+- [SCC, SCC+](scc.md)
 - [SegaPCM](segapcm.md)
-- [Seta/Allumer X1-010](x1-010.md)
+- [SM8521](sm8521.md)
+- [SN76489, SN76489A, SN76494, SN76496, SN94624, 8496, PSSJ](sms.md)
 - [SNES](snes.md)
-- [Atari 2600 (TIA)](tia.md)
 - [tildearrow Sound Unit](soundunit.md)
-- [TI SN76489](sms.md)
-- [Toshiba T6W28](t6w28.md)
+- [T6W28](t6w28.md)
+- [TED](ted.md)
+- [TIA](tia.md)
 - [VERA](vera.md)
-- [WonderSwan](wonderswan.md)
+- [VIC](vic20.md)
 - [Virtual Boy](virtual-boy.md)
-- [Yamaha YM2413 (OPLL)](opll.md)
-- [Yamaha OPL](opl.md)
-- [Yamaha YM2151](ym2151.md)
-- [Yamaha YM2203](ym2203.md)
-- [Yamaha YM2414 (OPZ)](opz.md)
-- [Yamaha YM2608](ym2608.md)
-- [Neo Geo/YM2610](ym2610.md)
-- [Taito Arcade/YM2610B](ym2610b.md)
-- [Yamaha YM2612](ym2612.md)
-- [Yamaha YMZ280B](ymz280b.md)
-- [ZX Spectrum Beeper](zxbeep.md)
-
-Furnace also reads .dmf files with the [Yamaha YMU759](ymu759.md) system, but...
+- [VRC6](vrc6.md)
+- [WonderSwan](wonderswan.md)
+- [X1-010](x1-010.md)
+- [(OPL) VRC7, Y8950, YM3526, YM3812, YMF262](opl.md)
+- [(OPLL) YM2413](opll.md)
+- [(OPZ) YM2414](opz.md)
+- [(OPM) YM2151](ym2152.md)
+- [(OPN) YM2203](ym2203.md)
+- [(OPNA) YM2608](ym2608.md)
+- [(OPNB) YM2610](ym2610.md)
+- [(OPNB) YM2610B](ym2610b.md)
+- [(OPN2) YM2612, YM3438](ym2612.md)
+- [YMU759 (MA-2)](ymu759.md)
```

Comment:
Don't kill the note. Restore.

`Furnace also reads .dmf files with the [Yamaha YMU759](ymu759.md) system, but...`

Reply:
And remove this entryyyyyyyyyy